db_cleanup_lambda/requirements.txt

```diff
@@ -0,0 +1,2 @@
+dcplib==1.4.0
+inflection==0.3.1
```

Comment:
why do you need `inflection` for the db_cleanup script?

Reply:
removed